src/browserLauncher.ts

```diff
@@ -97,9 +100,16 @@ async function _waitForWSEndpoint(browserProcess): Promise<string> {
     _addEventListener(rl, 'line', onLine);
 
     function onLine(line) {
+      const errorMatch = line.match(/Unable to open X display/);
```

Comment:
Is this the error you get on macOS and Windows too? Or does the error never happen there in the first place?

Reply:
I faced this issue when I tried to run a headful browser via `ssh`. The script just silently hang. I decided to add the specific error, and of cause It can be extended with other use cases.

Reply:
Extended the match logic with an array of regex, so the list can be adjusted with any other problems with launching Chrome.